### PR TITLE
SSX Monorepo Chores

### PR DESCRIPTION
**Pull request by skgbafa**
# Description
Adds a readme for the examples. Removed .npmrc in react example project. Moves changesets to devDependencies.

# Type
- [x] Chore

# Diligence Checklist
- [x] I have performed a self-review of my code
- [x] My changes generate no new warnings
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: examples/Readme.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
+# SSX Examples
+## Overview
+This directory contains examples of how to use the various packages in the SSX Monorepo. The examples are can be run standalone or together.
```
Comment (Juliano1612) [inline examples/Readme.md]:
```suggestion
This directory contains examples of how to use the various packages in the SSX Monorepo. The examples can be run standalone or together.
```

File: examples/Readme.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+## Running the examples
+To run the examples, you'll need to build the packages first. The best way to do that is in the root of the repo:
+```sh
+yarn ci
+# OR
+yarn && yarn build
+```
+
+Then, you can run the examples:
```
Comment (Juliano1612) [inline examples/Readme.md]:
```suggestion
Then, you can run ssx-test-dapp and ssx-test-express-api:
```

File: examples/Readme.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+```sh
+# From the root of the repo
+yarn run-examples
+# OR
+yarn test-dapp start
+yarn express-api start
+```
```
Comment (Juliano1612) [inline examples/Readme.md:30]: It would be nice to add a note at the end saying that ssx-test-dapp is compatible with all API examples.

**Review by slumericanB — APPROVED** [review]

File: examples/Readme.md (hunk 3 above)
Comment (samgbafa) [reply examples/Readme.md:30]: Done!

**Review by Juliano1612 — APPROVED** [review]
looks good to me

**Review by w4ll3 — APPROVED** [review]

**Pull request merged**